src/hubbleds/state.py

```diff
@@ -15,7 +22,45 @@ class LocalState:
     started: Reactive[bool] = dataclasses.field(default=Reactive(False))
     class_data_students: Reactive[list] = dataclasses.field(default=Reactive([]))
     class_data_info: Reactive[dict] = dataclasses.field(default=Reactive({}))
+    mc_scoring : Reactive[dict[str, MCScore]]  = dataclasses.field(default=Reactive({
+        # 'pro-dat1': {'tag': 'pro-dat1', 'score': 0.0, 'choice': 1, 'tries': 1, 'wrong_attempts': 0},
+        }))
 
 
 GLOBAL_STATE = GlobalState()
 LOCAL_STATE = LocalState()
+
+
+
+
+# create handlers for mc_radiogroup
+def onInitResponse(local_state , tag: str, set_score: callable = None): 
+    print("onInitResponse")
+    # print(tag not in component_state.mc_scoring.value.keys())
+    if tag not in local_state.mc_scoring.value.keys():
+        print("adding tag", tag)
+        mc_scoring = local_state.mc_scoring.value
```

Comment:
Initially I had made it a reactive class, but realized I was not using it reactively. Since the entire value can change, I just replace the whole `mc_score` for a given tag, so having the internal reactivity didn't seem to matter.  This solution seemed more compact. But it might be worth it to update just for the sake of a consistent style?